src/components/MyKiva/BadgesSection.vue

```diff
@@ -118,6 +122,10 @@ watch(selectedJourney, () => {
 		const badge = visibleBadges.value.find(b => b.id === selectedJourney.value);
 		if (badge) {
 			badgeClicked(badge);
+			if (props.nextStepSectionClicked) {
+				$kvTrackEvent('portfolio', 'view', 'Earn a badge', badge.challengeName, badge.level || 0);
```

Comment:
Reading the ticket, I think this is when a hero carousel item is shown (one event per journey card), not based on a click event. This component also already has a click event logged. Could you ask Shehaam when specifically this event should be logged?